### PR TITLE
Fixed Few Typos In Docs

### PR DESCRIPTION
**Pull request by saiteja13427**
**Description**:
Fixed a few typos in three of the docs.

**Motivation and Context**
Was going through the docs of this repository and found these issues so thought to fix them,

As these are minor corrections, I have directly raised a PR.

Hope it helps better the repository!!

### CODE REVIEW

**Review by justinchuby — DISMISSED** [review]
Thanks for your contribution!

**Comment by justinchuby** [comment]
Looks like the operators doc is generated

**Comment by saiteja13427** [comment]
@justinchuby shall I undo the changes in operators doc?

**Comment by justinchuby** [comment]
The best thing to do I think would be to figure out the source of the doc, and modify it instead.

**Comment by saiteja13427** [comment]
Alright, let me look for it and modify it.

**Comment by saiteja13427** [comment]
@justinchuby I have removed the changes to ContribOperators, tried looking for the descriptions for inputs, outputs and attributes but couldn't really find them, need help in locating them!

**Comment by justinchuby** [comment]
@saiteja13427 Try this https://sourcegraph.com/github.com/microsoft/onnxruntime/-/blob/onnxruntime/core/graph/contrib_ops/quantization_defs.cc?L172:38

**Review by justinchuby — CHANGES_REQUESTED** [review]
(just adding the state)

**Comment by saiteja13427** [comment]
@justinchuby I have checked the file you have shared, and that is perfectly correct. In fact, I have searched the whole project for the exact same typos, but those typos strangely exist only in the operators doc.

File: onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc [hunk 1]
```diff
@@ -465,4 +465,3 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
     OpSchema().FillUsing(ConvOpSchemaGenerator()));
 }  // namespace contrib
 }  // namespace onnxruntime
-
```
Comment (justinchuby) [inline onnxruntime/core/graph/contrib_ops/nhwc_schema_defs.cc:468]: Keep the new line

**Comment by saiteja13427** [comment]
Yeah, actually I wanted to compile but couldn't really figure out how to do that exactly, didn't find any guide as well in the repository.

Do we have any guide about how to compile or can you let me know how to do that.

**Comment by justinchuby** [comment]
Try this https://github.com/microsoft/onnxruntime/blob/master/tools/python/gen_contrib_doc.py

**Comment by justinchuby** [comment]
Here is a guide https://onnxruntime.ai/docs/build/inferencing.html but I don't think it is up to date. Try `./build.sh --config RelWithDebInfo --build_wheel --parallel --skip_tests` and pip install the wheel from the dist folder?

**Comment by saiteja13427** [comment]
Any idea why am I getting this error? @justinchuby 

image

**Comment by justinchuby** [comment]
I am not sure by just looking. Do does your machine have enough memory?

**Comment by saiteja13427** [comment]
Yeah I think this is a memory issue only

I am still getting an error even after disabling parallel

image

**Comment by justinchuby** [comment]
In this case I would just match the edits in the md file and let CI verify the change.

**Comment by saiteja13427** [comment]
Yeah, that is the only way I guess.

**Comment by justinchuby** [comment]
Thanks for your contribution. Closing it now due to inactivity. Please feel free to open new PRs for this change.

**Pull request closed without merging**